tools/docker/README.md

```diff
@@ -1,159 +1,79 @@
 # Using docker-compose for local development
 
-The docker-compose configuration present in this directory allows for a user to quickly setup all of chainlink's services to perform actions like integration tests, acceptance tests, and development across multiple services.
+The docker-compose configuration present in this directory allows to quickly run a local development environment.
 
 # Requirements
 
 - [docker-compose](https://docs.docker.com/compose/install/)
 
 # Using the compose script
 
-Inside the `chainlink/tools/docker` directory, there is a helper script that is included which should cover all cases of integration / acceptance / development needs acroos multiple services. To see a list of available commands, perform the following:
+Inside the `chainlink/tools/docker` directory, there is a helper script that is included
+To see a list of available commands, perform the following:
 
 ```sh
 cd tools/docker
 ./compose help
 ```
 
-## Examples
+## Env vars
 
-### Acceptance testing
-
-Acceptance can be accomplished by using the `acceptance` command.
+You can use the following env vars :
+- `GETH_MODE=true` to use geth instead of parity
+- `CHAIN_ID=<number>` to specify the chainID (default is 34055 for parity and 1337 for geth)
+- `HTTPURL=<url>` to specify the RPC node HTTP url (default is set if you use geth or parity)
+- `WSURL=<url>` to specify the RPC node WS url (default is set if you use geth or parity)
 
+for example :
 ```sh
-./compose acceptance
+CHAIN_ID=11155111 WSURL=wss://eth.sepolia HTTPURL=https://eth.sepolia ./compose dev
 ```
 
-- The chainlink node can be reached at `http://localhost:6688`
-
-Credentials for logging into the operator-ui can be found [here](../../tools/secrets/apicredentials)
-
-###
+## Dev
 
-### Doing local development on the core node
-
-Doing quick, iterative changes on the core codebase can still be achieved within the compose setup with the `cld` or `cldo` commands.
-The `cld` command will bring up the services that a chainlink node needs to connect to (parity/geth, postgres), and then attach the users terminal to a docker container containing the host's chainlink repository bind-mounted inside the container at `/usr/local/src/chainlink`. What this means is that any changes made within the host's repository will be synchronized to the container, and vice versa for changes made within the container at `/usr/local/src/chainlink`.
-
-This enables a user to make quick changes on either the container or the host, run `cldev` within the attached container, check the new behaviour of the re-built node, and repeat this process until the desired results are achieved.
+Will run one node with a postgres database and optionally by default a devnet RPC node that can be either geth or parity.
```

Comment:
does this "optionally" make sense? (I.e., it just runs one node with postgres db and a devnet RPC [parity by default]).

Reply:
similar for "./compose up"

Reply:
sure will commit the changes soon 👍